docs/articles/expensify-classic/getting-started/Invite.Members.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Invite Members
+description: Invite Members
+---
+# Overview
+
+To invite your employees in Expensify, simply add them as members to your policy.
+
+# How to Invite Employees to Expensify
+
+## Inviting Members Manually
+
+Navigate to **Settings > Policies > Group > *Policy Name* > People** - then click Invite and enter the invitee's email address.
```

Comment:
It looks like we don't want "Policy Name" to be bolded. Here's how it looks in the Google Doc:
<img width="650" alt="Screenshot 2023-09-27 at 8 12 52 AM" src="https://github.com/Expensify/App/assets/31285285/0b8ef358-520e-4046-8e8c-be49ade70d5f">

And here's how the markdown looks right now:

> Navigate to **Settings > Policies > Group > *Policy Name* > People** - then click Invite and enter the invitee's email address.

This should fix it!
```suggestion
Navigate to **Settings > Policies > Group >** *Policy Name* **> People** - then click Invite and enter the invitee's email address.
```